src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -1000,6 +1007,9 @@ bool clang_cpp_convertert::name_param_and_continue(
   std::string &name,
   exprt &param)
 {
+  if(!(id.empty() && name.empty()))
+    return false;
```

Comment:
Should this delegate back to the base class instead, like the default path below does?

Reply:
yes, it should